packages/common/src/intl/locales/en/distribution.json

```diff
@@ -83,5 +83,6 @@
   "messages.must-allocate-all-lines": "Cannot change the status until all lines have been allocated.",
   "messages.select-rows-to-delete": "Select rows to delete",
   "messages.shipment-saved": "Shipment saved 🥳",
-  "warning.nothing-to-supply": "Nothing left to supply!"
+  "warning.nothing-to-supply": "Nothing left to supply!",
+  "placeholder.search-by-name": "Search by name"
```

Comment:
I think this is used by more than just the `distribution` namespace and should be in `common`